### ISSUE TITLE
Timeshift does not work anymore

### ISSUE THREAD

**Issue by xpatos** [GoshPosh/grafana-meta-queries#122]
I have used meta-queries for a long time with ElasticSearch, and when updating to latest Grafana, our Timeshift graphs stopped working with the error "Cannot read property 'forEach' of undefined". I tried to update to latest version of Meta, but it does not help.

**Comment by Gauravshah** [comment]
have you tried using the latest version via github ?
we are working to get plugin signed and make 0.0.4 available but thats waiting on https://github.com/GoshPosh/grafana-meta-queries/issues/120

**Comment by xpatos** [comment]
Yes, I pulled the latest master from github, and restarted grafana.

**Comment by Gauravshah** [comment]
what grafana version are you using?
seems some issue with grafana 7.4 https://github.com/GoshPosh/grafana-meta-queries/issues/124

**Comment by xpatos** [comment]
Yes, I use Grafana v7.4.0 (c2203b9859)

**Comment by Gauravshah** [comment]
closing as duplicate of https://github.com/GoshPosh/grafana-meta-queries/issues/124
